index.html

```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
+            <div class="see-p">
+                <button type="button" class="see-project">See Project</button>
+            </div>
+        </div>
+        
+    </section>
+    <section class="first-project">
+        <div class="snapshot2"></div>
+        <div class="content-project1">
+            <h2>Grid Game</h2>
+            <div class="ali">
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>This is a sample from A website that teach us how to learn Grid in a game manner.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
+            <div class="see-p">
+                <button type="button" class="see-project">See Project</button>
+            </div>
+        </div>
+        
+    </section>
+    <section class="first-project">
+        <div class="snapshot3"></div>
+        <div class="content-project1">
+            <h2>Flex Box Game</h2>
+            <div class="ali">
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>This is website that teach flexbox in a game manner. They are just sample for my project.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
+            <div class="see-p">
+                <button type="button" class="see-project">See Project</button>
+            </div>
+        </div>
+        
+    </section>
+    <section class="about-me">
+        <div class="about">
+            <h2>About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social">
+                <p class="connect">LET’S CONNECT</p>
+                <ul class="social-ul">
+                    <li><a href=""><img src="images/Twitter.png" alt="Twitter"></a></li>
+                    <li><a href=""><img src="images/Github.png" alt="Github" class="sIcon"></a></li>
+                    <li><a href=""><img src="images/Linkedin-icon.png" alt="Linkedin" class="sIcon"></a></li>
+                </ul>
+                <div class="resume">
+                    <button type="button" class="resumee">Get my resume</button>
+                </div>
```

Comment:
- [ ] [REQUIRED] The button (e.g., "Get my resume") should follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1))

![image](https://user-images.githubusercontent.com/63915024/176977746-4372c7f9-0667-4317-9368-b34d69c7da14.png)